lib/config/recommended.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     'link-rel-noopener': 'strict',
     'link-href-attributes': true,
     'no-abstract-roles': true,
+    'no-args-paths': true,
```

Comment:
This may seem octane related, but ultimately if anyone is dabbling with `@glimmer/component` but haven't yet updated to use `octane` config they should still be guided away from `{{this.args.foo}}`.

Reply:
seems reasonable 👍